src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -38,6 +38,15 @@ public void RelativeSearchPath_Is_Null()
             Assert.Null(AppDomain.CurrentDomain.RelativeSearchPath);
         } 
 
+        [Fact]
+        public void TargetFrameworkTest()
+        {
+            RemoteInvoke(() => {
+                Assert.Contains(".NETCoreApp", AppContext.TargetFrameworkName);
+                return SuccessExitCode;
```

Comment:
I added overloads ofRemoteExec that take Action (no return code). You may be able to avoid returning one here...not sure which overload it picks